webpack.config.js

```diff
@@ -38,7 +40,11 @@ const config = {
       new BundleAnalyzerPlugin({
         analyzerMode: 'disabled',
         generateStatsFile: true
-      })
+      }),
+      new webpack.DefinePlugin({
+        APP_VERSION: JSON.stringify(JSON.parse(fs.readFileSync('./package.json')).version),
+        CIRCLE_SHA1: JSON.stringify(process.env.CIRCLE_SHA1 || '-')
```

Comment:
Let's please review this: is that all there is to the git hash? How can we test this?

https://circleci.com/docs/2.0/env-vars/